src/vue/backend/src/components/RelatedLinks.vue

```diff
@@ -0,0 +1,48 @@
+<template>
+    <idx-card>
+        <idx-card-header>{{ title }}</idx-card-header>
+        <idx-card-body>
+            <idx-list unstyled>
+                <idx-list-item v-for="link in links" :key="link.href">
+                    <a :href="link.href">{{ link.text }}</a>
+                </idx-list-item>
+            </idx-list>
+        </idx-card-body>
+    </idx-card>
+</template>
+
+<script>
+export default {
+    name: 'RelatedLinks',
+    data () {
+        return {
+            title: 'Related Links',
+            links: this.$store.state.relatedLinks
```

Comment:
Instead of using the store in the component, I think this should be a prop. The parent can make the determination of what links to pass it based on the store or its location, but this makes the component a little too opinionated.